test/common/components/CollapsiblePanel.unit.spec.jsx

```diff
@@ -1,33 +1,53 @@
 import React from 'react';
-import SkinDeep from 'skin-deep';
+import { shallow } from 'enzyme';
 import { expect } from 'chai';
+import sinon from 'sinon';
 
 import CollapsiblePanel from '../../../src/js/common/components/CollapsiblePanel';
 
+
 describe('<CollapsiblePanel>', () => {
   it('should render the correct panel header', () => {
-    const tree = SkinDeep.shallowRender(
-      <CollapsiblePanel
-        panelName={'Test panel'}/>
-    );
-
-    expect(tree.subTree('.usa-accordion-button').text()).to.contain('Test panel');
+    const testHeaderText = 'Test panel';
+    const wrapper = shallow(<CollapsiblePanel panelName={testHeaderText}/>);
+    const renderedHeaderText = wrapper.find('.usa-accordion-button').render().text();
+    expect(renderedHeaderText).to.equal(testHeaderText);
   });
 
   it('should handle toggling chapter', () => {
-    const tree = SkinDeep.shallowRender(
-      <CollapsiblePanel
-        panelName={'Test panel'}/>
-    );
+    const wrapper = shallow(<CollapsiblePanel panelName={'Test panel'}/>);
+
+    const toggleButton = wrapper.find('button');
+    expect(wrapper.find('.usa-accordion-content').length).to.equal(0);
+
+    toggleButton.simulate('click');
+    expect(wrapper.find('.usa-accordion-content').length).to.equal(1);
 
-    expect(tree.everySubTree('.usa-accordion-content')).to.be.empty;
+    toggleButton.simulate('click');
+    expect(wrapper.find('.usa-accordion-content').length).to.equal(0);
+  });
+
+  it('should default to open if startOpen prop is true', () => {
+    const wrapper = shallow(<CollapsiblePanel panelName={'Test'} startOpen/>);
+    const toggleButton = wrapper.find('button');
 
-    tree.getMountedInstance().toggleChapter();
+    expect(wrapper.find('.usa-accordion-content').length).to.equal(1);
 
-    expect(tree.everySubTree('.usa-accordion-content')).not.to.be.empty;
+    toggleButton.simulate('click');
+    expect(wrapper.find('.usa-accordion-content').length).to.equal(0);
+
+    toggleButton.simulate('click');
+    expect(wrapper.find('.usa-accordion-content').length).to.equal(1);
+  });
 
-    tree.getMountedInstance().toggleChapter();
+  it('should call scrollToTop on toggle open', () => {
+    const scrollSpy = sinon.spy();
+    CollapsiblePanel.prototype.scrollToTop = scrollSpy;
```

Comment:
Oooh, clever! Should probably reset it at the end of the test, though. Or maybe do this in a hook in case it fails?

The impact is minimal, I understand, but best practices, y'know?

Reply:
ahhh, that's a good point. Thanks for catching that!